src/main.js

```diff
@@ -198,6 +198,23 @@ function userCreatedPoster() {
   showMainPoster();
 }
 
+function savePoster() {
+  var save = {
+    image: posterImage.src,
+    title: posterTitle.innerText,
+    quote: posterQuote.innerText,
+  }
+  for (var i = 0; i < savedPosters.length; i++){
```

Comment:
Why are you using a for loop here?

When I run it, the object never makes it into the array. (How many times will this loop run when the array is empty?)

Reply:
Thank you for the leading question! The example below helps me visualize things more, too. Much appreciated! ✨